src/within.ts

```diff
@@ -15,7 +20,12 @@ export function within(instance: ReactTestInstance) {
     ...bindByPlaceholderTextQueries(instance),
     ...bindUnsafeByTypeQueries(instance),
     ...bindUnsafeByPropsQueries(instance),
-    ...a11yAPI(instance),
+    ...bindByLabelTextQueries(instance),
```

Comment:
nit: Can we move these bind calls above `unsafe` and have them sorted: label, role, a11yhint, a11y state, a11y states, a11y value?

Reply:
done